apps/assisted-installer-app/src/i18n.ts

```
@@ -0,0 +1,64 @@
+      }
+
+      return String(output);
+    },
+    escapeValue: false, // not needed for react as it escapes by default
+  },
+  react: {
+    useSuspense: true,
+    transSupportBasicHtmlNodes: true, // allow <br/> and simple html elements in translations
+  },
+  missingKeyHandler(lng, ns, key) {
+    if (lng instanceof Array) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)